rest/route/host_spawn.go

```diff
@@ -344,6 +367,142 @@ func (h *hostStartHandler) Run(ctx context.Context) gimlet.Responder {
 	return gimlet.NewJSONResponse(struct{}{})
 }
 
+////////////////////////////////////////////////////////////////////////
+//
+// POST /rest/v2/volumes
+
+const (
+	volumeDefaultProvider = evergreen.ProviderNameEc2OnDemand
+	volumeDefaultRegion   = evergreen.DefaultEC2Region
+)
+
+type createVolumeHandler struct {
+	sc data.Connector
+
+	Type string `json:"type"`
+	Size int    `json:"size"`
```

Comment:
I'm a little weird about serializing directly into the handler, and it seems like we'd probably want to have this be the same structure that we use to report the volume.